vmdb/app/models/miq_server/role_management.rb

```diff
@@ -153,7 +153,7 @@ def role=(val)
 
   def assign_role(server_role, priority = nil)
     server_role          = ServerRole.to_role(server_role)
-    assigned_server_role = self.assigned_server_roles.find_or_create_by_server_role_id(server_role.id)
+    assigned_server_role = assigned_server_roles.find_or_create_by(:server_role_id => server_role.id)
```

Comment:
@tenderlove find_or_create_by doesn't exist on rails 3.2.  I'm thinking we need to undo these changes on master and do them in rails4 branch, thoughts?